### DISCUSSION TITLE
H1 coil model

### DISCUSSION THREAD

**Discussion by moputradopol** [simnibs/simnibs#413] [2. General]
Hi all! 
I'm wondering if there is any update on the H1 coil model that can be downloaded. 
In the current documentation there is a note (Note 29.8.22) that states the H1 model is inaccurate due to a left-right mirror issue. 
Has this been resolved? 
Thanks in advance.
All the best

**Comment by diedrim** [comment]
We are still working on it, but we hope to have a fix ready within the next two months.
